### PR TITLE
Fix intermittency of tests

### PR DESCRIPTION
**Pull request by mauicv**
# What is this

`test_save_cfrl` intermittently fails as per [this](https://github.com/SeldonIO/alibi/issues/689) issue. (See issue notes there). I've set the `rtol` to 0, and the `atol` to `1e-06`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -76,6 +76,9 @@ target/
 # PyCharm
 .idea/
 
+# VSCode
+.vscode
```
Comment (ascillitoe) [inline .gitignore:80]: Not happy to see this 😉

File: alibi/tests/test_saving.py [hunk 2]
```diff
@@ -448,8 +448,8 @@ def test_save_cfrl(cfrl_explainer, rf_classifier, iris_data):
         assert exp0.meta == exp1.meta
 
         # cfrl is determinstic
-        assert_allclose(exp0.cf["X"].astype(np.float32), exp1.cf["X"].astype(np.float32), rtol=1e-06)
-        assert_allclose(exp0.cf["class"].astype(np.float32), exp1.cf["class"].astype(np.float32), rtol=1e-06)
+        assert_allclose(exp0.cf["X"].astype(np.float32), exp1.cf["X"].astype(np.float32), atol=1e-06, rtol=0)
```
Comment (ascillitoe) [inline alibi/tests/test_saving.py:451]: Just for context, what are the approx magnitudes of `exp1.cf["X"]` and `exp1.cf["class"]`? I ask in order to get an idea of how much larger `atol=1e-6` is compared to `rtol=1e-6`, given that `atol + rtol * abs(desired)` is the comparision in `allclose(actual, desired, ...)`.

**Review by ascillitoe — APPROVED** [review]
Assuming `atol` is not too big, I agree it makes sense to increase the tolerances for now so that CI passes, as long as we keep #689 open to revisit later...

File: alibi/tests/test_saving.py (hunk 2 above)
Comment (mauicv) [reply alibi/tests/test_saving.py:451]: between ~1.5 and ~-1.5 ish

Comment (ascillitoe) [reply alibi/tests/test_saving.py:451]: OK `atol` makes a lot of sense if `desired` is going negative, as I guess `rtol` gives an exceedingly small tolerance when `desired` hovers around zero?

**Pull request merged**